### DISCUSSION TITLE
Questions re update to newest version 2021.02.1

### DISCUSSION THREAD

**Discussion by fr9thjof** [tine20/tine20#7292] [General]
Hello All

We are happy user of Tine 2.0 for many years. It`s a great software. Thank you to everyone contributing.

We are currently on version 2019.02.07 and would like updating to the newest version 2021.02.1. Can we do it directly or do we need to update to other version in between before? Is there anything else we need to consider/do before going to 2021.02.1? Thank you very much.

Best,
Kjeld

**Comment by lab-at-nohl** [comment]
Have you considered the [Wiki page on updates](https://github.com/tine20/tine20/wiki/EN%3AUpdate-Howto)? Especially the update path.

Also double check that you have a **full Db backup**. And keep the files of your running version. Thus, you can return to the current installation if anything goes wrong. 

You need intermediate steps. Always take the highest minor version (20xx.xx.O) of the next major version (20xx.OO.x). 

 Best
Johannes

**Reply by fr9thjof** [reply]
Thank you Johannes. That helps a lot.
